docs/source/jobs.rst

```diff
@@ -53,35 +57,42 @@ Job-specific optional keys:
 Navigate
 --------
 
-This job type is a resource-intensive variant of "URL" to handle web pages
-requiring JavaScript in order to render the content to be monitored.
-
-The optional ``requests-html`` package must be installed to run "Navigate" jobs
-(see :ref:`dependencies`).
-
-.. code-block:: yaml
-
+The Navigate jobs are currently not working.
+
+They are intended to handle web pages requiring JavaScript in order to render
+the content to be monitored.
```

Comment:
As far as I could tell from my experiments in #407, navigate jobs are in fact working (at least for that use case there).

Reply:
I just checked. That case worked because Facebook loads events statically without using JavaScript on mobile pages. There's still no browser or JavaScript running with navigate/browser jobs. It's the same as using url jobs.

Reply:
The mobile Facebook page seems to indeed also contain the content, but there's still a different HTML content being loaded by `navigate`, so it does "something" (the CSS selectors in #407 don't work on the non-javascript page).